client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -1508,6 +1511,31 @@ private TxnSegments convert(CreateTxnResponse response) {
                 });
     }
 
+    @Override
+    public AsyncIterator<io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse> listCompletedTransactions(Stream stream) {
+        Exceptions.checkNotClosed(closed.get(), this);
+        Preconditions.checkNotNull(stream, "stream");
+        final long requestId = requestIdGenerator.get();
+        long traceId = LoggerHelpers.traceEnter(log, "listCompletedTransactions", stream, requestId);
+
+        try {
+            final Function<ContinuationToken, CompletableFuture<Map.Entry<ContinuationToken, Collection<io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse>>>> function =
```

Comment:
`io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse` can be imported and there should be a line-break

Reply:
Done as suggested.